server/claim/claim.service.ts

```diff
@@ -38,7 +38,6 @@ export class ClaimService {
             .sort({ _id: order })
             .lean();
         return Promise.all(
-            /** Get lastest revision for each claim */
             claims.map(async (claim) => {
```

Comment:
Here's a place where we forgot to update with the new `this._getClaim` functionality.

Reply:
I didn't understand